src/main.c

```diff
@@ -73,6 +73,9 @@ main() {
   int nfds;
   int udevfd;
 
+  /* init libusb */
+  usb_init();
```

Comment:
Sorry if this is obvious, but where is usb_init() defined? If this call wasn't there before, is it truly needed now?